docs/install.md

```diff
@@ -705,6 +706,10 @@ gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/pullrequest-init
 gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/webhook
 ```
 
+## Verify Tekton Resources
+
+[Trusted Resources](./trusted-resources.md) is a feature to verify Tekton Tasks and Pipelines. The current version of feature supports `v1beta1` `Task` and `Pipeline`. For more details please take a look at [TEP-0091](https://github.com/tektoncd/community/blob/main/teps/0091-trusted-resources.md).
```

Comment:
I wouldn't refer to the TEPs at all in the user facing docs; users should be able to understand the features using the docs and not need to look at the TEPs at all imo. The commit message and pull request can be a good place to refer to the TEP so the linkage is still available.

Reply:
sure! I will remove the link in docs

Reply:
The reason I linked it here is because to my understanding, docs readers are not only users but also developers/contributors. A link here may help them quickly direct to the design doc. We also include the `trusted-resources.md` link in this section and this is not to delegate the docs to teps. 

Besides, we also have something similar in this doc:
https://github.com/tektoncd/pipeline/blob/main/docs/install.md#configuring-pipelineresource-storage